src/components/referrals/ReferralsComponentLabel.vue

```diff
@@ -119,4 +119,79 @@
             height: 3rem;
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
```

Comment:
as above I dont think is it good solution

Reply:
jak wyzej